src/figuresClasses.ts

```diff
@@ -1,11 +1,70 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Стороны не образуют треугольник');
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Circle implements Figure {
+  shape: 'circle' = 'circle';
 
-export class Triangle implements Figure {}
+  constructor(
+    public color: string,
+    public radius: number,
+  ) {
+    if (radius <= 0) {
+      throw new Error('Радиус круга должен быть больше 0');
+    }
+  }
 
-export class Circle implements Figure {}
+  getArea(): number {
+    const area = Math.PI * this.radius ** 2;
 
-export class Rectangle implements Figure {}
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Rectangle implements Figure {
+  shape: 'rectangle' = 'rectangle';
+
+  constructor(
+    public color: string,
+    public width: number,
+    public height: number,
+  ) {
+    if (width <= 0 || height <= 0) {
+      throw new Error('Стороны квадрата должны быть положительными');
```

Comment:
[CODE STYLE] - If you throw an error try to provide its detailed description in error message. The error message 'Стороны квадрата должны быть положительными' should be in English and more descriptive, e.g., 'Rectangle sides must be positive numbers.' Also, the term 'квадрата' (square) is used instead of 'rectangle', which might be a translation error.